extensions/ql-vscode/src/view/model-editor/HiddenMethodsRow.tsx

```diff
@@ -0,0 +1,30 @@
+import {
+  VSCodeDataGridCell,
+  VSCodeDataGridRow,
+} from "@vscode/webview-ui-toolkit/react";
+import * as React from "react";
+import { styled } from "styled-components";
+
+const HiddenMethodsCell = styled(VSCodeDataGridCell)`
+  text-align: center;
+`;
+
+interface Props {
+  numHiddenMethods: number;
+  someMethodsAreVisible: boolean;
+}
+
+export function HiddenMethodsRow(props: Props) {
```

Comment:
I think this matches with more of the components:

```suggestion
export function HiddenMethodsRow({ numHiddenMethods, someMethodsAreVisible }: Props) {
```